src/lib/mock-fetch.ts

```diff
@@ -16,9 +16,9 @@ export async function mockFetch(
   url: string,
   options: RequestInit = {},
 ): Promise<Response> {
-  //throw new Error("ERROR APPEARAD");
   return new Promise((resolve) => {
     setTimeout(() => {
+      //rej (new Error("ERROR APPEARAD"));
```

Comment:
**note:** Make sure to remove those comments once you merge. Is this a leftover btw? Maybe I'm just silly and did something wrong but when i wanted to check the error state throwing errors didn't work here, I had to go to the recommendation query itself and just throw an error there before returning the data

Reply:
to throw an error, you have to add reject method to Promise
```suggestion
return new Promise((resolve, rej) => {
    setTimeout(() => {
      rej (new Error("ERROR APPEARAD"));
```

Reply:
Thanks for clarifying